components/grid/Col.razor.cs

```diff
@@ -59,6 +59,8 @@ public partial class Col : AntDomComponentBase
 
         [Parameter]
         public OneOf<int, EmbeddedProperty> Xxl { get; set; }
+        [Parameter]
+        public AntLableAlignType LableAlignType { get; set; } = AntLableAlignType.Right;
```

Comment:
Please rename the property to `LabelAlign` as well as ant design  react.

Reply:
@ElderJames